### PR TITLE
Class Version 4

### PR DESCRIPTION
**Pull request by GIT-REPOSITORY-TEST**
Please review the changes

### CODE REVIEW

File: zsales_class_git.clas.abap [hunk 1]
```diff
@@ -12,8 +12,9 @@ CLASS zsales_class_git DEFINITION
         matnr TYPE matnr,
         charg TYPE charg_d,
         werks TYPE werks_d,
```
Comment (cpuvanes) [inline zsales_class_git.clas.abap:14]: Please include material description and storage location

File: zsales_class_git.clas.abap [hunk 2]
```diff
@@ -12,8 +12,9 @@ CLASS zsales_class_git DEFINITION
...
-        matkl TYPE matkl,
+*        matkl TYPE matkl,   "New change V3 - Branch 3
         erdat TYPE erdat,    "New Change V2 - Branch 2
+        ernam TYPE ernam,    "New Change V3 - Branch 3
       END OF gty_vbap .
     TYPES:
       gtt_t_vbap TYPE TABLE OF gty_vbap .
```
Comment (cpuvanes) [inline zsales_class_git.clas.abap]: Please specify table type explicitly(STANDARD/SORTED)

**Review by cpuvanes — CHANGES_REQUESTED** [review]

File: zsales_class_git.clas.abap [hunk 3]
```diff
@@ -12,8 +12,9 @@ CLASS zsales_class_git DEFINITION
         matnr TYPE matnr,
         charg TYPE charg_d,
         werks TYPE werks_d,
-        matkl TYPE matkl,
+*        matkl TYPE matkl,   "New change V3 - Branch 3
```
Comment (Sakthy-g) [inline zsales_class_git.clas.abap:15]: As per FD - ok

**Review by Sakthy-g — COMMENTED** [review]
ok

**Comment by cpuvanes** [comment]
I have completed my review

**Comment by GIT-REPOSITORY-TEST** [comment]
Changes fixed. Please review again

**Review by GIT-REPOSITORY-TEST — COMMENTED** [review]
ok

**Pull request merged**